tests/helpers.py

```diff
@@ -107,11 +107,16 @@ def calref_from_image(input_image):
 
     for step in corr_lookup:
         # Not all images have the CORR step and it is not always on.
-        if (step not in hdr) or (hdr[step].strip().upper() != 'PERFORM'):
+        # Disable the PERFORM check in order to download ALL reference files
+        # associated with a calibration step present in the header
+        if (step not in hdr):
             continue
 
-        ref_files += ref_from_image(input_image, corr_lookup[step])
+        single_step_files = ref_from_image(input_image, corr_lookup[step])
+        if (single_step_files):
+            ref_files += single_step_files
 
+    print(ref_files)
```

Comment:
Might want a "debug" option for the function to toggle the printing on/off, unless you accidentally left this in and was meaning to remove it before merge.

Reply:
Thanks for the comment, but the print is deliberate.  The tests themselves (IMO) should be a bit more informative, particularly tests which perform a lot of tasks which is not this test, and log more information even under successful circumstances.

Reply:
If you do want the tests to produce more information, it might be better to use the `logging` package rather than `print`.  This way, the information can be more easily filtered and turned on or off as needed by the user.

Reply:
For reference: https://docs.pytest.org/en/latest/logging.html